Api/OrderLogRepositoryInterface.php

```diff
@@ -16,6 +16,12 @@ interface OrderLogRepositoryInterface
      */
     public function getList(SearchCriteriaInterface $searchCriteria) : array;
 
+    /**
+     * @return int
+     * @throws \Exception
+     */
+    public function getCount() : int;
```

Comment:
this is a breaking change, so we will need to bump version number, which is OK, but please add [!!!] at the beginning of the commit message to make clear this change is breaking